web/components/feed/feed-comments.tsx

```diff
@@ -224,15 +233,17 @@ export function getMostRecentCommentableBet(
     .pop()
 }
 
-function CommentStatus(props: { contract: Contract; outcome: string }) {
-  const { contract, outcome } = props
+function CommentStatus(props: {
+  contract: Contract
+  outcome: string
+  prob?: number
+}) {
+  const { contract, outcome, prob } = props
   return (
     <>
       {' betting '}
```

Comment:
Maybe use " betting on" instead? When the FR answer is number it looks like you're betting that amount.